src/parameter.cpp

```diff
@@ -21,65 +21,70 @@
 using namespace std;
 
 #define ERRORLOG cerr << "ERROR in Parameter: "
-#define TRACELOG if (_verbose) cout << "TRACE in Parameter: "
+#define TRACELOG \
+  if (_verbose)  \
+  cout << "TRACE in Parameter: "
 
-namespace acap_runtime {
-
-// Initialize Parameter Service
-bool Parameter::Init(const bool verbose) {
-  _verbose = verbose;
-  TRACELOG << "Init" << endl;
-  return true;
-}
+namespace acap_runtime
+{
+  Parameter::~Parameter()
+  {
+    if (ax_parameter != NULL)
+      ax_parameter_free(ax_parameter);
+  }
 
-// Logic and data behind the server's behavior.
-Status Parameter::GetValues(ServerContext* context,
-    ServerReaderWriter<Response, Request>* stream) {
+  // Initialize Parameter Service
+  bool Parameter::Init(const bool verbose)
```

Comment:
See my comments in https://github.com/AxisCommunications/acap-runtime/pull/80/files about constructor instead of two-phase initialization.